packages/nube-devtools/src/manifest.ts

```
@@ -0,0 +1,38 @@
+import { defineManifest } from '@crxjs/vite-plugin'
+import packageData from '../package.json'
+
+//@ts-ignore
+const isDev = process.env.NODE_ENV == 'development'
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)